pymc3/distributions/timeseries.py

```diff
@@ -176,6 +184,75 @@ def logp(self, value):
         return at.sum(innov_like) + at.sum(init_like)
 
 
+class ARMARV(RandomVariable):
+    name = "ARMA"
+    ndim_supp = 0
+    ndims_params = [1, 1, 0, 0]
+    dtype = "floatX"
+    _print_name = ("ARMA", "\\operatorname{ARMA}")
+
+    def __call__(self, phi=[], theta=[], loc=[0.0], scale=[1.0], size=100, **kwargs) -> TensorVariable:
+        return super().__call__(phi, theta, loc, scale, size=size, **kwargs)
+
+    @classmethod
+    def rng_fn(
+            cls,
+            rng: np.random.default_rng(),
+            phi: np.ndarray,
+            theta: np.ndarray,
+            loc: np.ndarray,
+            scale: np.ndarray,
+            size: Tuple[int, ...],
+    ) -> np.ndarray:
+        phi = np.r_[1, -phi]
+        theta = np.r_[1, theta]
+        x = rng.normal(loc=loc, scale=scale, size=size)
+        return signal.lfilter(theta, phi, x, axis=0)
```

Comment:
Maybe a high level comment what the lfilter does?

Reply:
will add all this, thanks a ton @michaelosthege